spikes/data/dataservice/app/DataAccessLayer/InMemoryDal.cs

```diff
@@ -146,6 +146,40 @@ public Task<IEnumerable<Actor>> GetActorsAsync(ActorQueryParameters actorQueryPa
             });
         }
 
+        /// <summary>
+        /// Get actor IDs by search criteria
+        /// </summary>
+        /// <param name="actorQueryParameters">search criteria</param>
+        /// <returns>Cosmos select statement</returns>
+        public string GetActorIds(ActorQueryParameters actorQueryParameters)
+        {
+            List<Actor> list;
+
+            if (actorQueryParameters == null)
+            {
+                list = GetActors(string.Empty, 0, 100);
+            }
+            else
+            {
+                list = GetActors(actorQueryParameters.Q, actorQueryParameters.GetOffset(), actorQueryParameters.PageSize);
+            }
+
+            string ids = string.Empty;
+
+            if (list != null && list.Count > 0)
+            {
+                foreach (Actor a in list)
+                {
+                    ids += $"'{a.ActorId}',";
+                }
+
+                string sql = "select m.id, m.partitionKey, m.actorId, m.type, m.name, m.birthYear, m.deathYear, m.profession, m.textSearch, m.movies from m where m.id in ({0}) order by m.textSearch ASC";
```

Comment:
Should the sql statements used be converted to `const` or `static readonly`?

Reply:
good catch - resolved